{"name": "test_forgot_password_003", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_TIMED_OUT\n  (Session info: chrome=135.0.7049.85)\nStacktrace:\n\tGetHandleVerifier [0x00007FF784B65335+78597]\n\tGetHandleVerifier [0x00007FF784B65390+78688]\n\t(No symbol) [0x00007FF7849191AA]\n\t(No symbol) [0x00007FF784915BA0]\n\t(No symbol) [0x00007FF784906849]\n\t(No symbol) [0x00007FF7849085A8]\n\t(No symbol) [0x00007FF784906B56]\n\t(No symbol) [0x00007FF7849065D6]\n\t(No symbol) [0x00007FF78490629A]\n\t(No symbol) [0x00007FF784903F4A]\n\t(No symbol) [0x00007FF78490471C]\n\t(No symbol) [0x00007FF78491D07A]\n\t(No symbol) [0x00007FF7849C002E]\n\t(No symbol) [0x00007FF7849970EA]\n\t(No symbol) [0x00007FF7849BF2BB]\n\t(No symbol) [0x00007FF784996EC3]\n\t(No symbol) [0x00007FF7849603F8]\n\t(No symbol) [0x00007FF784961163]\n\tGetHandleVerifier [0x00007FF784E0EEED+2870973]\n\tGetHandleVerifier [0x00007FF784E09698+2848360]\n\tGetHandleVerifier [0x00007FF784E26973+2967875]\n\tGetHandleVerifier [0x00007FF784B8017A+188746]\n\tGetHandleVerifier [0x00007FF784B8845F+222255]\n\tGetHandleVerifier [0x00007FF784B6D2B4+111236]\n\tGetHandleVerifier [0x00007FF784B6D462+111666]\n\tGetHandleVerifier [0x00007FF784B53589+5465]\n\tBaseThreadInitThunk [0x00007FFADADDE8D7+23]\n\tRtlUserThreadStart [0x00007FFADBEB14FC+44]", "trace": "self = <testCases.test_forgot_password.Test_004_Forgot_Password object at 0x000001A4D6C076F0>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"007bdcf42215255f061f438eee70dbac\")>\n\n    @pytest.mark.sanity\n    def test_forgot_password_003(self, setup):\n        self.driver = setup\n        self.logger.info(\"************ Test Forgot Password 003 Start ************\")\n>       self.driver.get(self.baseURL)\n\ntestCases\\test_forgot_password.py:70: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:454: in get\n    self.execute(Command.GET, {\"url\": url})\n.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py:429: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x000001A4D71F8980>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_TIMED_OUT\\\\n...84B53589+5465]\\\\n\\\\tBaseThreadInitThunk [0x00007FFADADDE8D7+23]\\\\n\\\\tRtlUserThreadStart [0x00007FFADBEB14FC+44]\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_TIMED_OUT\nE         (Session info: chrome=135.0.7049.85)\nE       Stacktrace:\nE       \tGetHandleVerifier [0x00007FF784B65335+78597]\nE       \tGetHandleVerifier [0x00007FF784B65390+78688]\nE       \t(No symbol) [0x00007FF7849191AA]\nE       \t(No symbol) [0x00007FF784915BA0]\nE       \t(No symbol) [0x00007FF784906849]\nE       \t(No symbol) [0x00007FF7849085A8]\nE       \t(No symbol) [0x00007FF784906B56]\nE       \t(No symbol) [0x00007FF7849065D6]\nE       \t(No symbol) [0x00007FF78490629A]\nE       \t(No symbol) [0x00007FF784903F4A]\nE       \t(No symbol) [0x00007FF78490471C]\nE       \t(No symbol) [0x00007FF78491D07A]\nE       \t(No symbol) [0x00007FF7849C002E]\nE       \t(No symbol) [0x00007FF7849970EA]\nE       \t(No symbol) [0x00007FF7849BF2BB]\nE       \t(No symbol) [0x00007FF784996EC3]\nE       \t(No symbol) [0x00007FF7849603F8]\nE       \t(No symbol) [0x00007FF784961163]\nE       \tGetHandleVerifier [0x00007FF784E0EEED+2870973]\nE       \tGetHandleVerifier [0x00007FF784E09698+2848360]\nE       \tGetHandleVerifier [0x00007FF784E26973+2967875]\nE       \tGetHandleVerifier [0x00007FF784B8017A+188746]\nE       \tGetHandleVerifier [0x00007FF784B8845F+222255]\nE       \tGetHandleVerifier [0x00007FF784B6D2B4+111236]\nE       \tGetHandleVerifier [0x00007FF784B6D462+111666]\nE       \tGetHandleVerifier [0x00007FF784B53589+5465]\nE       \tBaseThreadInitThunk [0x00007FFADADDE8D7+23]\nE       \tRtlUserThreadStart [0x00007FFADBEB14FC+44]\n\n.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py:232: WebDriverException"}, "attachments": [{"name": "log", "source": "0497f516-55c5-414b-9b09-8b087cc33042-attachment.txt", "type": "text/plain"}], "start": 1744711683450, "stop": 1744711726532, "uuid": "38f584bb-ad85-41e5-9d42-03d87a9e92e9", "historyId": "057cc185fac5f30ac8d521d58d39d91f", "testCaseId": "057cc185fac5f30ac8d521d58d39d91f", "fullName": "test_forgot_password.Test_004_Forgot_Password#test_forgot_password_003", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "suite", "value": "test_forgot_password"}, {"name": "subSuite", "value": "Test_004_Forgot_Password"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "4620-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_forgot_password"}]}